_posts/2017-06-28-Openshift-an-introduction.md

```diff
@@ -0,0 +1,241 @@
+This means that your team spends less time in the weeds and keeping the lights on, and more time being innovative and productive. 
+
+### Opensource
+<span class="image left small"><img class="p-image" alt="opensource" src="/img/openshift-an introduction/opensource.png"></span>
+There are multiple versions of openshift (Spoiler: it's gonna be the next topic in this blog post) but they are all based on Openshift Origin.
+Origin provides an open source application container platform. All source code for the Origin project is available under the Apache License (Version 2.0) on [GitHub](https://github.com/openshift)
+
+# Openshift landscape
+
+There are a few different openshift releases depending what you need.
+As of this writing the openshift landscape looks like this.
+
+<p>
+  <img class="image fit" alt="openshiftlandscape" src="/img/openshift-an introduction/openshiftlandscape.png">
+</p>
+
+### Openshift origin
+It's the upstream community project used in openshift Online, Openshift dedicated and Openshift container Platform.
+It's build around docker and kubernetes cluster management.
+Origin is augmented by application lifecycle management functionality and DevOps tooling.
+Origin updates as often as open source developers contribute via git sometimes as often as several times per week. Here you get the new feature the quickest but at the cost of stability.
+
+### Openshift container platform
+Formaly known as openshift enterprise.
+It's the  platform software  to deploy and manage openshift on your own  infrastructure of choice.
+It integrates with Red Hat Enterprise Linux 6 and is tested via Red Hat's QA process in order to offer a stable, supportable product with may be important for enterprises.
+
+### Openshift dedicated
+Openshift dedicated is the latest offering of openshift.
+It's OpenShift 3 hosted on AWS and maintained by Red Hat but it is dedicated to you
+
+### Openshift online
+OpenShift Online is managed by Red Hat’s OpenShift operations team, and quickstart templates enable developers to push code with one click, helping to avoid the intricacies of application provisioning.
+You can view it as openshift delivered as a SAAS(software as a service)
+
+# Benefits for developers
+Before I show you how easy openshift is for a developer, let me quickly explain Source to image (S2I) to make life even more easier.
+
+
+Let's show how easy your life can be with the following image
+
+> __source to image__ is a toolkit and workflow that creates a deployable docker image based on your source code and add it to the image registry. You don't even need a docker file anymore.It combines source code with a corresponding builder image from the integrated docker registry
```

Comment:
Source-to-Image
Docker file anymore. It ... (space after the period)
... Docker registry ...